lib/oauth2client.js

```diff
@@ -75,6 +75,10 @@ OAuth2Client.prototype._createTokenUrl = function () {
   parameters.msaproxy = true;
```

Comment:
but now for b2c and convergence, even in prod, it needs this parameter, then should I change it to be extra parameter and pass it?